ts/storage-todos.ts

```diff
@@ -0,0 +1,11 @@
+import { Todo } from "./types.js";
+import { storage } from "./storage.js";
+import { demoTodos } from "./demoTodos.js";
+
+let todos = storage.get<Todo[]>("todos") ?? demoTodos;
+
+const modifyStorageTodos = (newArray: Todo[]): void => {
```

Comment:
I think it's overkill to reassign variable just to get the fresh localstorage state in it. This function can be used to provide `any` value to todos function, which is not correct. `todos` should always have fresh localstorage state, so rename function to `syncStorage` and just do:

```
todos = storage.get<Todo[]>('todos')
```